modules/gdscript/editor_templates/CharacterBody2D/basic_movement.gd

```diff
@@ -2,8 +2,9 @@
 
 extends _BASE_
 
-const SPEED: float = 300.0
-const JUMP_VELOCITY: float = -400.0
+
+const SPEED = 300.0
+const JUMP_VELOCITY = -400.0
```

Comment:
I think the templates are supposed to use type annotations so they can be conditionally removed based on the user configuration.

Reply:
As far as I'm aware, constants infer typing automatically. Therefore, making explicitly stating their types pointless.

Reply:
No idea, but in that case I guess it makes sense.